### ISSUE TITLE
Unable to make webpack build fail even when shell script exits non 0

### ISSUE THREAD

**Issue by mneil** [s00d/webpack-shell-plugin-next#47]
Super basic example that was addressed in #8 but seems to have regressed or maybe doesn't work in my case.

```
new WebpackShellPluginNext({
    onAfterDone: {
	scripts: ["exit 1"],
	blocking: true,
	parallel: false,
    },
}),
```

Running `npx webpack` I expect the exit code to be non 0. However, webpack build still succeeds. Any suggestions on how to fail the webpack process?

```
$ npx webpack
$ echo $?
0
```

**Comment by mneil** [comment]
Adding the `safe` option to true to use exec instead of spawn works as intended. Would be nice to work with spawn as well and track the exit code of that process but good enough for now.

```
new WebpackShellPluginNext({
    onAfterDone: {
	scripts: ["exit 1"],
	blocking: true,
	parallel: false,
    },
    safe: true,
}),
```

Thanks!

**Comment by retyui** [comment]
@mneil thank you! faced the same issue

**Comment by s00d** [comment]
Hi, I’ve released the 2.3.3 update — give it a try, it should help.
